osmaxx-py/osmaxx/excerptexport/static/excerptexport/scripts/formModeSwitcher.js

```diff
@@ -26,8 +26,13 @@
             formPartsSwitcher.value = formModeParam;
         }
 
+        // IE may throw an error here "Object doesn't support this action".
+        // It seems IE won't trigger a change event on select boxes.
+        // I didn't find a solution to get the supported actions,
+        // so the error is still there: Fuck you IE.
+        // Note: The switcher is working regardless.
```

Comment:
I guess this means it works **on IE, too**? If so, you might want to spell that out.